src/views/Expenses.js

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from 'react';
+import AddExpensesForm from '../components/AddExpensesForm';
+import ExpensesList from '../components/ExpensesList';
+
+const Expenses = () => {
+  const [expenses, setExpenses] = useState([]);
+
+  // Run on initial render only
+  useEffect(() => {
+    // Fetch expense items
+    const token = process.env.REACT_APP_MOSTASH_API_KEY;
+    fetch(`http://pi.motine.de:12305/items.json?stash=${token}&kind=expense`)
```

Comment:
do we want to use the `then` or `async` paradigm in this project?

Reply:
I am bug supporter of `async/await`

Reply:
together with try/catch blocks

Reply:
I don't have strong feelings either way. `async/await` then?

@GiulianoMontisEvernest do you think we need try/catch here?

Reply:
it is good practice to call `fetch()` inside of a try/catch block. 
We do not have to, but it makes things easier especially when handling errors.

Reply:
network requests can always fail (think: you are sitting in the U-Bahn). i think adding error handling would be good.